monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js

```diff
@@ -20,12 +22,16 @@ function RunOptions(props) {
 
   useEffect(() => {
     if (initialized === false) {
-      authComponent.authFetch('/api')
-        .then(res => res.json())
-        .then(res => {
-          setIps([res['ip_addresses']][0]);
-          setInitialized(true);
+      authComponent.authFetch(CONFIG_URL)
+      .then(res => res.json())
+      .then(res => {
+        let commandServers = res.configuration.internal.island_server.command_servers;
```

Comment:
We really need to test this somehow. If we change configuration structure or rename a single field, this will silently break. Next quarter we should look into UI unit tests, but for now I suggest you add a unit test in the back end, which verifies that this path indeed exists.